wdl/Vapor.wdl

```diff
@@ -20,15 +20,18 @@ workflow Vapor {
     String sv_base_mini_docker
     String sv_pipeline_docker
 
-    RuntimeAttr? runtime_attr_vapor 
+    RuntimeAttr? runtime_attr_vapor
     RuntimeAttr? runtime_attr_bcf2vcf
     RuntimeAttr? runtime_attr_vcf2bed
     RuntimeAttr? runtime_attr_split_vcf
     RuntimeAttr? runtime_attr_concat_beds
     RuntimeAttr? runtime_attr_LocalizeCram
+
+    Boolean save_plots  # New input to control whether plots are final output
```

Comment:
```suggestion
    Boolean save_plots  # Control whether plots are final output
```
You don't need to specify that it's a new input - in a year it won't be new! I would also move this up - probably between sample_id and ref_fasta. We like to group inputs together, and this isn't a runtime attribute so it should be higher up. You can leave NONE_FILE_ where it is though to discourage people from trying to provide it a value.

Reply:
Thank you for that clarification and organization suggestion. These changes were made.